ProcessLib/GroundwaterFlow/GroundwaterFlowFEM.h

```diff
@@ -74,10 +75,16 @@ class LocalAssemblerData
         assert(local_matrix_size == ShapeFunction::NPOINTS * NUM_NODAL_DOF);
     }
 
-    void assemble(double const /*t*/, std::vector<double> const& local_x) override
+    void assemble(std::size_t const id,
```

Comment:
If  `id` means an element ID, is it really needed to pass it here? `Element` object is already stored in this class.